### PR TITLE
[FIX](map)insert into doris table with array/map type by local tvf

### PR DESCRIPTION
**Pull request by amorynan**
## Proposed changes

Issue Number: close #xxx
fix read map type data when insert into doris table by local tvf 

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by amorynan** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 48.36 seconds
 stream load tsv:          510 seconds loaded 74807831229 Bytes, about 139 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.3 seconds inserted 10000000 Rows, about 341K ops/s
 storage size: 17161972099 Bytes

File: regression-test/suites/external_table_p0/tvf/test_local_tvf_with_complex_type_insertinto_doris.groovy [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            "backend_id" = "${be_id}",
+            "format" = "orc");"""
+
+    qt_sql """
+        insert into ${table_name} select * from local (
+            "file_path" = "${dataFilePath}/comp.orc",
+            "backend_id" = "${be_id}",
+             "format" = "orc");"""
+
+    qt_sql """ select * from ${table_name}; """
```
Comment (yiguolei) [inline regression-test/suites/external_table_p0/tvf/test_local_tvf_with_complex_type_insertinto_doris.groovy:59]: add order by

File: regression-test/suites/external_table_p0/tvf/test_local_tvf_with_complex_type_insertinto_doris.groovy [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    qt_sql """
+        insert into ${table_name} select * from local(
+            "file_path" = "${dataFilePath}/comp.parquet",
+            "backend_id" = "${be_id}",
+            "format" = "parquet"); """
+
+    qt_sql_count """ select count(*) from ${table_name} """
+
+    qt_sql """ select * from ${table_name} """
```
Comment (yiguolei) [inline regression-test/suites/external_table_p0/tvf/test_local_tvf_with_complex_type_insertinto_doris.groovy:75]: add order by

**Comment by amorynan** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 46.03 seconds
 stream load tsv:          513 seconds loaded 74807831229 Bytes, about 139 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.3 seconds inserted 10000000 Rows, about 341K ops/s
 storage size: 17162114321 Bytes

**Review by xiaokang — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**